content/monetization/partner-program-api/index.mdx

```diff
@@ -1714,6 +1720,87 @@ curl --request GET \
 
 </Section>
 
+<Section>
+
+<Text>
+
+## Invite License Operators
+
+`POST /v2/partners/solutions/licenses/livechat/<license_id>/operators` - invite operators to a LiveChat license
+
+#### Headers
+
+- `Authorization` - **required** - `Bearer <YOUR_API_TOKEN>`
+
+#### Path Parameters
+
+- `<license_id>` - **required** - license ID
+
+#### Body
+
+- `operators` - **required** - list of the **users** you want to invite to the license.
+
+Struct of **user** object:
+
+- `email` - **required** - user email
+- `role` - **required** - `administrator` or `agent` ([learn more about the roles](https://www.livechatinc.com/help/livechat-roles-owner-administrators-and-agents/))
+
+#### Response
+
+- `200 - OK` - success (could be partly, check for `status_code` in response body), see the example on the right
```

Comment:
What does it mean that success could be partial?

Reply:
It means that some users might be invited and some not - e.g. because of already taken email